src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="submit"
+      class="form"
+    >
```

Comment:
The `action` attribute of the form tag is supposed to contain the URL where the form data is sent to. In your case, `submit` is not a valid URL. If you don't have a specific URL to send the form data to, you can just leave this attribute out.